<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crowley's Commander</string>
  </property>
  <property name="windowIcon">
   <iconset resource="my_res.qrc">
    <normaloff>:/icons/icons/crowleys_commander.ico</normaloff>:/icons/icons/crowleys_commander.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">* {color: #fff;}
QWidget#mainWidget, QWidget#mainBar,
QWidget#leftPanel, QWidget#rightPanel,
QTreeView {
	background: #333;
	color: #fff;
	border: 1px solid #444;
}
QMenuBar {spacing: 0;}
QMenuBar::item {
	padding: 3px 10px;
}
QMenu::item {
	padding: 5px 15px;
}
QMenuBar::item:selected, QMenu::item:selected {
	background: #222;
	border: 1px solid #333;
}
QMenu:hover, QAction:hover {
	background: #292929;
}

QPushButton {
	border: 1px solid #333;
	background: #222;
	padding: 5px 7px;
	border-radius: 3px;
}
QPushButton:hover {
	background: #292929;
}
QPushButton:pressed {
	background: #222;
}

QComboBox {
	background: #222;
	border: 1px solid #444;
	border-radius: 3px;
}
QComboBox:hover {
	background: #292929;
}
QComboBox::drop-down {
	border: none;
	background: transparent;
}
QComboBox::down-arrow {
	width: 12px;
	height: 12px;
	margin-right: 6px;
	image: url(:/icons/icons/arrow.png);
}

QWidget#leftPanel, QWidget#rightPanel {
	border: 1px solid #444;
}

QWidget#statusDiskLeft, QWidget#statusDiskRight {
	border-top: 1px solid #444;
	border-bottom: 1px solid #444;
}

QWidget#bottomButtons {
	border-top: 1px solid #444;
}

QWidget#mainWidget QLineEdit {
	background: #ea4242;
	padding: 4px 0;
	border-radius: 0;
	color: #222;
}

QWidget#mainWidget QLineEdit:focus {
	background: #fa5252;
	color: #000;
}

QTabWidget::tab-bar	{
	alignment: left;
}

QTabBar::tab {
	background: #222;
	padding: 6px 10px;
	border-top-left-radius: 5px;
	border-top-right-radius: 5px;
	border: 1px solid #444;
}
QTabBar::tab:hover {
	background: #292929;
}
QTabBar::tab:selected {
	background: #fa5252;
	color: #000;
}
QTabBar::tab:selected:hover {
	background: #ff5757;
	color: #000;
}

QTreeView::item {
	border-bottom: 1px solid #444;
	padding: 4px 0;
}
QTreeView::item:hover {
	background: #393939;
}
QTreeView::item:selected {
	color: #fa5252;
	background: transparent;
}
QHeaderView::section {
	background: #222;
	border-bottom: 1px solid #555;
	border-right: 1px solid #444;
	height: 20px;
	margin-top: 5px;
	margin-left: 5px;
}
QHeaderView::section:hover {
	background: #292929;
}

QHeaderView::down-arrow {
    image: url(:icons/icons/arrow.png);
	width: 12px;
	height: 12px;
	margin-right: 8px;
}
QHeaderView::up-arrow {
    image: url(:icons/icons/arrow_up.png);
	width: 12px;
	height: 12px;
	margin-right: 8px;
}</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="mainBar" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="leftPanel" native="true">
         <layout class="QVBoxLayout" name="leftPanelLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="diskButtonsLeft" native="true">
            <layout class="QHBoxLayout" name="diskButtonsLeftLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="statusDiskLeft" native="true">
            <layout class="QHBoxLayout" name="statusDiskLeftLayout">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QComboBox" name="diskListLeft">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>20</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="diskNameLeft">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="diskSizeLeft">
               <property name="text">
                <string>diskSizeLeft</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="rightPanel" native="true">
         <layout class="QVBoxLayout" name="rightPanelLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="diskButtonsRight" native="true">
            <layout class="QHBoxLayout" name="diskButtonsRightLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="statusDiskRight" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="statusDiskRightLayout">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QComboBox" name="diskListRight">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>20</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="diskNameRight">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="diskSizeRight">
               <property name="text">
                <string>diskSizeLeft</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="bottomButtons" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="editBtn">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>F4 Edit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="copyBtn">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>F5 Copy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moveBtn">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>F6 Move</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="folderBtn">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>F7 New Folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteBtn">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>F8 Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>24</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew_File"/>
    <addaction name="actionOpen_selected_file"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_with"/>
    <addaction name="actionRename"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionRemove_permanently"/>
    <addaction name="actionCreate_Folder"/>
    <addaction name="actionCreate_Shortcut"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuSelection">
    <property name="title">
     <string>Selection</string>
    </property>
    <addaction name="actionSelect_file"/>
    <addaction name="actionSelect_all"/>
    <addaction name="actionRemove_selection"/>
   </widget>
   <widget class="QMenu" name="menuTabs">
    <property name="title">
     <string>&amp;Tabs</string>
    </property>
    <addaction name="actionCreate_a_new_tab"/>
    <addaction name="actionClose_this_tab"/>
    <addaction name="separator"/>
    <addaction name="actionClose_all_tabs"/>
    <addaction name="actionSwitch_to_the_next_tab"/>
    <addaction name="separator"/>
    <addaction name="actionSwitch_to_the_previous_tab"/>
    <addaction name="actionOpen_the_folder_in_the_new_tab"/>
    <addaction name="actionOpen_the_folder_in_the_new_tab_in_another_bar"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionFile_search"/>
    <addaction name="actionShow_Hide_hidden_files"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSelection"/>
   <addaction name="menuTabs"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionNew_File">
   <property name="text">
    <string>&amp;New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_selected_file">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Open selected file</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionOpen_with">
   <property name="text">
    <string>Open &amp;with</string>
   </property>
   <property name="shortcut">
    <string>Shift+Return</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>&amp;Rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>R&amp;emove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionRemove_permanently">
   <property name="text">
    <string>Re&amp;move permanently</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="actionCreate_Folder">
   <property name="text">
    <string>&amp;Create Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionCreate_Shortcut">
   <property name="text">
    <string>Create &amp;Shortcut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>C&amp;ut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Cop&amp;y</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelect_file">
   <property name="text">
    <string>&amp;Select file</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>Select &amp;all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRemove_selection">
   <property name="text">
    <string>&amp;Remove selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionCreate_a_new_tab">
   <property name="text">
    <string>&amp;Create a new tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionClose_this_tab">
   <property name="text">
    <string>C&amp;lose this tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionClose_all_tabs">
   <property name="text">
    <string>Close &amp;all tabs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionSwitch_to_the_next_tab">
   <property name="text">
    <string>&amp;Switch to the next tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionSwitch_to_the_previous_tab">
   <property name="text">
    <string>Switch to the &amp;previous tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionOpen_the_folder_in_the_new_tab">
   <property name="text">
    <string>&amp;Open the folder in the new tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionOpen_the_folder_in_the_new_tab_in_another_bar">
   <property name="text">
    <string>Open &amp;the folder in the new tab in another bar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Up</string>
   </property>
  </action>
  <action name="actionFile_search">
   <property name="text">
    <string>&amp;File search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionShow_Hide_hidden_files">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Show/Hide hidden files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="my_res.qrc"/>
 </resources>
 <connections/>
</ui>
